src/components/LocationMarker/LocationMarker.test.js

```diff
@@ -53,14 +70,17 @@ describe('components/LocationMarker', () => {
     ])
     expect(Leaflet.Circle.prototype.setRadius).toHaveBeenCalledWith(accuracy)
 
+    act(() => {
+      fireEvent.click(screen.queryByTestId('gpsButton'))
+    })
+
```

Comment:
No need to wrap a call to `fireEvent` in `act`. Also, the preferred way of handling user interaction, is to use `userEvent` instead of `fireEvent`.

Reply:
✅